interface/orders/types.php

```diff
@@ -17,13 +17,13 @@
 // item ID to select) will be provided, and maintenance may also be permitted.
 
 $popup = empty($_GET['popup']) ? 0 : 1;
-$order = formData('order', 'G') + 0;
-$labid = formData('labid', 'G') + 0;
+$order = isset($_GET['order']) ? $_GET['order'] : 0;
+$labid = isset($_GET['labid']) ? $_GET['labid'] : 0;
 
 // If Save was clicked, set the result, close the window and exit.
 //
 if ($popup && $_POST['form_save']) {
-    $form_order = formData('form_order') + 0;
+    $form_order = isset($_GET['form_order']) ? $_GET['form_order'] : 0;
```

Comment:
add 0

Reply:
oh, it just hit me that adding 0 to these values is potentially the problem you are solving :)

Reply:
looks like the is_numeric you used below is a good route to go for this stuff

Reply:
Nope, you are correct with the +0, it should be there to ensure an int or float and not string. Will fix above.... Previously the PHP parser only sees a string function being set to a numeric type so warns. The conditional allows parser to resolve to the correct var typing so, is happy but, we still need to ensure int with +0.